common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -104,8 +104,16 @@ status_function() {
     ps -aux | grep local_bus
 }
 
+check_function() {
```

Comment:
I think it is better to add this into the status function

Reply:
OPERATIONAL in the main agent doesn't mean the radio agents are operational - the status check should first indeed check that the main agent is operational, and then grep beerocks_agent_wlan0.log to see it got to a working state - for now, in DUMMY mode, we know the son slave is functional if we see `ACTION_AP_MANAGER_HOSTAP_CSA_NOTIFICATION`